ramesh/data/Lab-1-Charts-Basics.ipynb

```diff
@@ -0,0 +1,530 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    import mplfinance as mpf
</pre><p>Add imports at the top cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>